core/integration/client_generator_test.go

```diff
@@ -476,11 +478,11 @@ export class Test {
 						WithNewFile("index.ts", `import { connection, dag } from "@dagger.io/client"
 
 async function main() {
-    await connection(async () => {
-      const res = await dag.test().hello()
+  await connection(async () => {
+    const res = await dag.test().hello()
 
-      console.log("result:", res)
-    })
+    console.log("result:", res)
+  })
```

Comment:
nit: fix your editor's indent settings

Reply:
The string snippet are not handled by my editor sadly haha so I have to do it manually every time! Good catch

Reply:
Here I'm actually fixing a indent issue from a previous PR, normally we use 2 space, not 4 :)

Reply:
nit: fix someone else's editor indent settings lol